releasenotes/notes/remove-deprecated-pulse-code-d98fc32cf385221a.yaml

```diff
@@ -0,0 +1,13 @@
+---
+upgrade:
+  - |
+    Remove from Qiskit Pulse:
+
+      - ``SamplePulse``
+      - ``ConstantPulse``
+      - ``get_sample_pulse``
+      - ``pulse.pulse_lib``
+      - ``Instruction.command``
+      - ``Acquire.acquires``, ``mem_slots``, ``reg_slots``
```

Comment:
Could you maybe make this a table and list the old way and the new alternative. Something like: https://github.com/Qiskit/qiskit-terra/blob/master/releasenotes/notes/0.16/3927-0d94adeac9299540.yaml#L8

It's good to include what people should be using now in the upgrade notes. While we also had in the deprecation note and the warnings when people look at this it's nice to say "This thing X that you shouldn't have been using because we told you it is going away has now been removed. If you're still using X despite our warnings you should use Y instead."

If not we can do it in the 0.17.0 release notes pr.